### PR TITLE
Added instructions on how to setup PostgreSQL in macOS

### PR DESCRIPTION
**Pull request by kitsiosk**
Will double-check them tomorrow using a second macOS machine

### CODE REVIEW

**Comment by jc00ke** [comment]
For macOS I'd also throw out https://postgresapp.com/ as a super easy way to get Postgres installed.

**Comment by kitsiosk** [comment]
> For macOS I'd also throw out https://postgresapp.com/ as a super easy way to get Postgres installed.

Thanks, I added this option, together with motivation for using postgresql, and brief explanation of server-client model

**Comment by kitsiosk** [comment]
Update: I tested both installation options on a second Mac machine, and made the required adjustments. Next, I will add simple queries to get started

File: db/penguins.sql [hunk 1]
```diff
@@ -0,0 +1,374 @@
+BEGIN TRANSACTION;
```
Comment (gvwilson) [inline db/penguins.sql:1]: the raw data is in `misc/penguins.csv` - is it possible to load directly into PSQL from there or do you need the full SQL dump?

**Pull request merged**